docs/units/equivalencies.rst

```diff
@@ -413,6 +413,24 @@ scaling:
     <Quantity 11.2254902 mag>
 
 
+Temperature Equivalency
+-----------------------
+
+The :func:`~astropy.units.temperature` equivalency allows conversion between the temperature units; degree Celsius, Fahrenheit and Kelvin.
+Example:
```

Comment:
Maybe rephrase the end to say 'between the Celsius, Fahrenheit and Kelvin temperature units. For example::' (and make sure you put two colons (::) at the end so that the code block appears correctly.

Reply:
Ok done.
But I don't understand why travis-ci and circleci failed.

Reply:
circle CI has failed with something unrelated, as you didn't start from the latest master.

the travis failure is due to the differences in float representations, try to add ``# doctest: +FLOAT_CMP`` to the end of L428

Reply:
Alright, I get it a bit. I should add just like this?
`<Quantity 32. deg_F>    # doctest: +FLOAT_CMP`

Reply:
I messed up the line numbers. It should be on L427, the one that is executed rather than the output.

Reply:
Okay, I changed it. Thank you @bsipocz